qiita_pet/support_files/config_portal.txt

```diff
@@ -0,0 +1,41 @@
+# ---------- Base information for the website ----------
```

Comment:
That would be good, but is not necessary as part of this PR ... though
cheers if you do do that!

On (Jun-23-15|10:23), Joshua Shorenstein wrote:
> > @@ -0,0 +1,41 @@
> > +# ---------- Base information for the website ----------
> 
> Can do. Should I change the base config file to .cfg instead of .txt as well?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1275/files#r33067032